prototypes/templates.lua

```diff
@@ -129,4 +129,21 @@ if data.raw.item["se-space-transport-belt"] then
   }
 end
 
+-- RandomFactorioThings and PlutoniumEnergy support
+-- note: RFT doesn't add belts and other plutonium machinery without PE mod
+if mods["RandomFactorioThings"] then
```

Comment:
Honestly, I don't understand why to check for items and not for mod. For me, it makes sense that if mod is loaded, then there **will** be these items.

Reply:
The comment says that RFT will not add the items we care about if PE is not installed, so just checking for RFT is not enough. If a player runs RFT without PE, this code would cause Miniloader to crash.

Having some other mod be present or absent is not the only thing that might be important. Bob’s Logistics adds _different_ items depending on mod settings.

It’s dangerous to try to copy the logic that another mod uses to decide what items to add. It might change in the future. (What is RFT adds a setting that causes it to not add the items we need even if PE is installed?) So it’s safest for the future to just check for what we actually care about: do the items we reference in our recipes and the technologies we depend on as prerequisites actually exist?

Reply:
In the code, there's a check for PE after checking if RFT is present.
And also I, as an author of RFT, assure that RFT will **never** have an option for disabling content.

Reply:
I see. You are also the author of PE. That makes guarantees easier to offer. :)